cypress/integration/login.js

```diff
@@ -12,16 +12,17 @@ describe('Sourcebooks login', function() {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Jevgenija Malomuž"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
         cy.get('[aria-label="User"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
+        cy.get('.calendar--today').contains('24')

```

Comment:
I would recommend to add this assert after line " cy.get('.calendar').should('be.visible')" because after that you will be able to see calendar. Cypress is smart enough to wait for element to appear, but other frameworks are not